huaweicloud/resource_huaweicloud_swr_organization.go

```diff
@@ -0,0 +1,123 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/swr/v2/namespaces"
+)
+
+func resourceSWROrganization() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSWROrganizationCreate,
+		Read:   resourceSWROrganizationRead,
+		Delete: resourceSWROrganizationDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		//request and response parameters
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"creator": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"permission": {
```

Comment:
shall we add `registry` attribute which users can push images
the format of registry is  `swr.{region}.myhuaweicloud.com/{name}`

Reply:
added a login_server url instead as what Azure does.